### PR TITLE
Channel close implemented

### PR DESCRIPTION
**Pull request by rdmitr**

### CODE REVIEW

File: src/greenlight_alby_client.rs [hunk 1]
```diff
@@ -775,6 +775,45 @@ impl From<cln::WithdrawResponse> for WithdrawResponse {
...
+    pub destination: Option<String>,
+    pub fee_negotiation_step: Option<String>,
+    pub force_lease_closed: Option<bool>,
+}
+
+impl From<CloseRequest> for cln::CloseRequest {
+    fn from(req: CloseRequest) -> Self {
+        cln::CloseRequest {
+            id: req.id,
+            unilateraltimeout: req.unilateral_timeout,
```
Comment (rolznz) [inline src/greenlight_alby_client.rs:791]:
```suggestion
            unilateral_timeout: req.unilateral_timeout,
```

**Review by rolznz — APPROVED** [review]
utACK

File: src/greenlight_alby_client.rs (hunk 1 above)
Comment (rdmitr) [reply src/greenlight_alby_client.rs:791]: I agree it's kinda ugly but this field's name comes from Greenlight — we cannot change it

Comment (rolznz) [reply src/greenlight_alby_client.rs:791]: ah, thanks!

**Pull request merged**